pyproject.toml

```
@@ -22,7 +30,33 @@
     "scipy>=1.9.2,<2;python_version>='3.10'",
 ]
 
-development_requires = [
+[project.urls]
+"Source Code"= "https://github.com/sdv-dev/Copulas/"
+"Issue Tracker" = "https://github.com/sdv-dev/Copulas/issues"
```

Do we have keep this changelog up to date? We could add it here
```suggestion
"Issue Tracker" = "https://github.com/sdv-dev/Copulas/issues"
"Changes" = "https://github.com/sdv-dev/Copulas/blob/main/HISTORY.md"
```